.github/workflows/benchmark_nightly_gpu.yml

```
@@ -0,0 +1,25 @@
+name: Benchmark torchserve gpu nightly
+
+on:
+  # run every day at 2:15am
+  schedule:
+    - cron:  '15 02 * * *'
```

try this for a test first so we know things work end to end before we commit code
```suggestion
on:
  push:
    branches:
      - issue_1454
```